sql/src/test/java/io/crate/metadata/doc/array/ArrayMapperTest.java

```diff
@@ -528,4 +531,42 @@ public void testParseDynamicNullArray() throws Exception {
         assertThat(doc.docs().get(0).getField("new_array_field"), is(nullValue()));
         assertThat(mapper.mappers().smartNameFieldMapper("new_array_field"), is(nullValue()));
     }
+
+    @Test
+    public void testCopyToFieldsOfInnerMapping() throws Exception {
+        String mapping = XContentFactory.jsonBuilder()
```

Comment:
Maybe surround with `@formatter:off` and `@formatter:on` to make sure autoformat doesn't remove indentations